docs/src/hashing.md

```diff
@@ -0,0 +1,59 @@
+```@meta
+CurrentModule = Kmers
+DocTestSetup = quote
+    using BioSequences
+    using Test
+    using Kmers
+end
+```
+
+!!! warning
+    The value of hashes are guaranteed to be reproducible for a given version
+    of Kmers.jl and Julia, but may __change__ in new minor versions of Julia
+    or Kmers.jl
```

Comment:
Yeah agreed. Let's revisit this and reconsider stabilizing the hash value some time after Kmers 1.0. It's tricky, because there are different tradeoffs for different hash function, so for really performance critical code, users should probably be able to choose their own hash function.